src/java.desktop/unix/native/libsplashscreen/splashscreen_sys.c

```diff
@@ -710,7 +712,9 @@ SplashScreenThread(void *param) {
 //    pthread_setspecific(key, splash);
 
     SplashLock(splash);
-    pipe(splash->controlpipe);
+    if (pipe(splash->controlpipe)) {
+        fprintf(stderr, "Error creating pipe for splash screen control: %s\n", strerror(errno));
```

Comment:
what happens to the process if this call fails ?
Can it continue ?
You can check by commenting out the call and see what happens when you run an app on Linux that uses splashScreen ? SwingSet2 would be sufficient.
And assuming it is a continuable error, can you do the print only in a DEBUG build ?